src/utils/route-paths.ts

```diff
@@ -28,14 +28,14 @@ export const VALIDATORS = '/validators';
 
 export const WALLET = 'https://wallet.camino.network/';
 
-export const DOCS = 'https://docs.camino.foundation/';
+export const DOCS = 'https://docs.camino.network/';
 
 export const DISCORD = 'https://discord.gg/camino';
 
 export const TWITTER = 'https://twitter.com/caminonetwork';
```

Comment:
This URL is wrong. It leads to a suspended account. The correct one should be https://twitter.com/camino_network/
We should update it while we're at it.

Reply:
Done